### PR TITLE
tiflash: add scale.md doc

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

Add `scale.md` doc for TiFlash.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

**If you select two or more versions from above**, to trigger the bot to cherry-pick this PR to your desired release version branch(es), you **must** add corresponding labels such as **needs-cherry-pick-4.0**, **needs-cherry-pick-3.1**, **needs-cherry-pick-3.0**, and **needs-cherry-pick-2.1**.

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/2603
- Other reference link(s):

### CODE REVIEW

File: reference/tiflash/scale.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+## Scale in a TiFlash node
+
+The following example shows how to scale in a TiFlash node if you deploy TiFlash on the `192.168.1.1` node.
+
+> **Note:**
+>
+> The offline process described in this section does not delete the data on the offline node. If you need to take the node online again, delete the data manually.
+
+1. Take the node to be scaled in offline. See [Take TiFlash nodes offline](/reference/tiflash/maintain.md#take-tiflash-nodes-offline) for details.
```
Comment (ran-huang) [inline reference/tiflash/scale.md:98]: The reference anchor is not finalized since the translation of `maintain.md` is not finished yet.

Comment (yikeke) [reply reference/tiflash/scale.md:98]: FYI @toutdesuite

**Comment by yikeke** [comment]
> This applies only to 3.1.
> 4.0 will have different content.

The 4.0 changes might refer to https://github.com/pingcap/docs-cn/pull/2641/.

File: reference/tiflash/scale.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+# Scale the TiFlash Cluster
+
+This document describes how to scale in and out nodes in the TiFlash cluster.
+
+## Scale out a TiFlash node
+
+The following example shows how to scale out a TiFlash node if you deploy TiFlash on the `192.168.1.1` node.
+
+1. Edit the `inventory.ini` file, and add the TiFlash node information:
```
Comment (TomShawn) [inline reference/tiflash/scale.md:15]:
```suggestion
1. Edit the `inventory.ini` file to add the TiFlash node information:
```

File: reference/tiflash/scale.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+    Currently, this feature only supports IP but not domain name.
+
+    {{< copyable "" >}}
+
+    ```ini
+    [tiflash_servers]
+    192.168.1.1
+    ```
+
+2. Edit the `hosts.ini` file, and add the node information:
```
Comment (TomShawn) [inline reference/tiflash/scale.md:26]:
```suggestion
2. Edit the `hosts.ini` file to add the node information:
```

File: reference/tiflash/scale.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+
+    ```shell
+    ansible-playbook rolling_update_monitor.yml --tags=prometheus
+    ```
+
+7. Access the monitoring platform using your browser, and view the status of the cluster and the new node.
+
+## Scale in a TiFlash node
+
+The following example shows how to scale in a TiFlash node if you deploy TiFlash on the `192.168.1.1` node.
```
Comment (TomShawn) [inline reference/tiflash/scale.md:92]:
```suggestion
The following example shows how to scale in a TiFlash node if you stop the TiFlash service on the `192.168.1.1` node.
```

**Comment by ran-huang** [comment]
@TomShawn PTAL again~

File: TOC.md [hunk 5]
```diff
@@ -308,6 +308,7 @@
     - [Deploy a TiFlash Cluster](/reference/tiflash/deploy.md)
     - [Use TiFlash](/reference/tiflash/use-tiflash.md)
     - [Monitor TiFlash](/reference/tiflash/monitor.md)
+    - [Scale](/reference/tiflash/scale.md)
```
Comment (TomShawn) [inline TOC.md:311]:
```suggestion
    - [Scale TiFlash](/reference/tiflash/scale.md)
```

File: reference/tiflash/scale.md [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+
+5. Start the new node:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    ansible-playbook start.yml -l 192.168.1.1
+    ```
+
+6. Update the configuration of Prometheus and restart:
```
Comment (TomShawn) [inline reference/tiflash/scale.md:80]: Please check with the author @ilovesoup `restart` what.

File: reference/tiflash/scale.md [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    ansible-playbook stop.yml -t tiflash -l 192.168.1.1
+    ```
+
+4. Edit the `inventory.ini` and `hosts.ini` files to remove the node information.
+
+5. Update the configuration of Prometheus and restart:
```
Comment (TomShawn) [inline reference/tiflash/scale.md:120]: ditto, restart what.

File: reference/tiflash/scale.md (hunk 6 above)
Comment (ran-huang) [reply reference/tiflash/scale.md:80]: Confirmed: restart `Prometheus`

**Comment by ran-huang** [comment]
@TomShawn PTAL

**Comment by yikeke** [comment]
I will merge this PR after https://github.com/pingcap/docs/pull/2189/ is merged.

File: reference/tiflash/scale.md (hunk 1 above)
Comment (toutdesuite) [reply reference/tiflash/scale.md:98]: @ran-huang The anchor is decided: `#take-a-tiflash-node-down`.

**Comment by yikeke** [comment]
A dead link is introduced by this PR but it will be fixed by https://github.com/pingcap/docs/pull/2189 soon.

**Pull request merged**